tests/pr_test.go

```diff
@@ -498,3 +499,29 @@ func TestDependencyPermutations(t *testing.T) {
 	err := options.RunAddonPermutationTest()
 	assert.NoError(t, err, "Dependency permutation test should not fail")
 }
+
+func TestQuickstartSchematics(t *testing.T) {
+	t.Parallel()
+
+	options := testschematic.TestSchematicOptionsDefault(&testschematic.TestSchematicOptions{
+		Testing: t,
+		Prefix:  "vsi-qs",
+		TarIncludePatterns: []string{
+			"*.tf",
+			quickStartConfigFlavorDir + "/*.tf",
+		},
+		TemplateFolder:         quickStartConfigFlavorDir,
+		Tags:                   []string{"vsi-qs-da"},
+		DeleteWorkspaceOnFail:  false,
```

Comment:
I think you should set this to true, we usually only need the workspace to stay if we need to debug the logs, otherwise we want to clean it up.